src/main/java/pl/betoncraft/betonquest/conversation/ConversationData.java

```diff
@@ -96,7 +97,14 @@ public ConversationData(ConfigPackage pack, String name) throws InstructionParse
 		String rawStartingOptions = pack.getString("conversations." + name + ".first");
 		String stop = pack.getString("conversations." + name + ".stop");
 		blockMovement = stop != null && stop.equalsIgnoreCase("true");
+		convIO = pack.getString("conversations." + name + ".conversationIO");
+		if (convIO == null) {
+			convIO = BetonQuest.getInstance().getConfig().getString("default_conversation_IO");
+		}
 		// check if all data is valid (or at least exist)
+		if (BetonQuest.getInstance().getConvIO(convIO) == null) {
+			throw new InstructionParseException("Conversation IO is not registered!");
```

Comment:
It'd be good to display the `convIO` variable in this message, so the user immediately knows what mistake he made.
👍 for adding the loading-time check here 😄